### ISSUE TITLE
Shows hunter pet ability information instead of NPC creature abilities on tamable beasts.

### ISSUE THREAD

**Issue by ButteredGroove** [icheatatlan/ClassicBestiary#5]
For instance, [Young Black Bear](https://www.wowhead.com/classic/npc=1128/young-black-bear#abilities;mode:normal) shows [Growl](https://www.wowhead.com/classic/spell=6796/growl) as an ability even though the creature does not cast it.

Can you detect and blacklist hunter pet abilities from beasts when generating the abilities database?

**Comment by icheatatlan** [comment]
Yes, there is a large WIP database update that I haven't quite polished up enough to release yet which will remove hunter pet abilities. Will be published some time before official HC servers release.
